packages/core/src/cache/classes/Surface.ts

```diff
@@ -43,6 +43,18 @@ export class Surface {
     return this.polys;
   }
 
+  public setColor(color: Point3): void {
```

Comment:
Don't we have a color type?  If it has the same shape, using a different name allows changing things int he future and finding everywhere using it.